packages/react-jsx-highstock-datepickers/src/index.js

```diff
@@ -58,6 +58,10 @@ class DateRangePickers extends Component {
       fromLabel,
       toLabel
     };
+    //For input field value
+		moment.locale(locale, {
+			...langOpts
+		});
```

Comment:
Would you mind using spaces instead of tabs for consistency with the rest of the codebase?